modules/charset/escape.go

```diff
@@ -65,10 +65,6 @@ func EscapeControlStringReader(reader io.Reader, writer io.Writer, locale transl
 			}
 			break
 		}
-		if err := streamer.SelfClosingTag("br"); err != nil {
-			streamer.escaped.HasError = true
-			return streamer.escaped, err
```

Comment:
Hmm, but won't that be erroneous for things that are not inside `<pre>` tags?

Reply:
That's not meant for how it's used at the moment, but for how it might be used.
People might assume that newlines will also be persisted in the HTML, so the least thing to do is clarify that that isn't the case in the method comment.